app/src/pages/playground/Playground.tsx

```diff
@@ -123,6 +123,7 @@ const playgroundInputOutputPanelContentCSS = css`
 
 function PlaygroundContent() {
   const instances = usePlaygroundContext((state) => state.instances);
+  const inputs = usePlaygroundContext((state) => state.input);
```

Comment:
I suspect there will be a better way to extract this data given that the shape of the input object changes based on whether we are in dataset or manual mode.